fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/DropPrimaryKey.java

```diff
@@ -34,8 +35,18 @@ public DropPrimaryKey(String schemaName, String tableName) {
 
     @Override
     public void run(IDatabaseTranslator translator, Connection c) {
+        // ought to be doing this via an adapter, which hides the differences between databases
         final String qname = DataDefinitionUtil.getQualifiedName(this.schemaName, this.tableName);
-        final String ddl = "ALTER TABLE " + qname + " DROP PRIMARY KEY";
+        final String ddl;
```

Comment:
I don't usually use final on local variables; was a little surprised we can use it here and initialize it later...shows what I know :-)